### DISCUSSION TITLE
ハンズオンの質問

### DISCUSSION THREAD

**Discussion by Ikuyadeu** [vscodejp/handson-hello-vscode-extension#5] [Q&A]
ハンズオンでわからないことがあればこちらでご質問ください．

**Comment by fuguman** [comment]
Hello Language Server ProtocolのLaunch Clientのところは、Windows版だとDebugマーク▽のところでしょうか？微妙にインタフェースが異なるようでそのあたりで遭難しました。

**Comment by fuguman** [comment]
![image](https://user-images.githubusercontent.com/3297212/142710783-81ef74bf-b8d3-4da2-bd25-1b9c80516948.png

**Comment by dafujii** [comment]
私も Windows 環境ですがステータスバーからデバッグには行けないようでした。
左側の「実行とデバッグ」を開いて「Client＋Server」を選ぶといいです。
![image](https://user-images.githubusercontent.com/41186511/142710780-85785678-2e6f-4ad3-b1af-01807ecc7fcd.png)

**Comment by fuguman** [comment]
![image](https://user-images.githubusercontent.com/3297212/142710836-7ef47970-a602-45b5-a660-a4156927cbf2.png)
これですね。
有難うございます。

**Reply by Ikuyadeu** [reply]
ご報告いただき，ありがとうございました．
後ほどドキュメントを更新いたします．

**Comment by nafuka11** [comment]
MacOS でも、同じくステータスバーからデバッグに飛べないようでした。
「実行とデバッグ」から実行できました。
![2021-11-20 11-11-01](https://user-images.githubusercontent.com/42476527/142710903-718ab02b-688c-4580-92cd-51e904b65cc7.png)
一度実行した後は、ステータスバーに表示されるようです。
![2021-11-20 11-13-56](https://user-images.githubusercontent.com/42476527/142710939-311ca89f-f9df-4d2f-b49b-c487bd01d7d6.png)

**Reply by mochan-tk** [reply]
そうなんですね！知らなかった....
ありがとうございます！

**Comment by 74th** [comment]
ステータスバーへのデバッグの表示は、設定`"debug.showInStatusBar": "always",`で常に表示されるようになるようです。
初期状態は、初めてデバッグ実行をしたときに表示されるようになる、という設定になっていました。

**Comment by keides2** [comment]
飛び入りがここでお尋ねしてよいのかわかりませんが、よろしければ教えていただけますか？
「ドキュメントを編集しよう」で、コンパイルエラーが発生しています。
![image](https://github.com/vscodejp/handson-hello-vscode-extension/assets/18452832/063c4a4d-3b24-42f9-9e14-2e104faad461)
